frontend/e-buddy/src/components/chatbot/ChatPreview.tsx

```
@@ -48,184 +241,499 @@ export const ChatPreview = ({ customization, onClose }: ChatPreviewProps) => {
+      <div
         className="p-3 flex justify-between items-center cursor-pointer"
         onClick={() => setIsExpanded(!isExpanded)}
-        style={{ 
-          backgroundColor: customization.primaryColor || '#9b87f5',
-          color: 'white'
+        style={{
+          backgroundColor: customization.primaryColor || "#9b87f5",
+          color: "white",
         }}
       >
         <div className="flex items-center space-x-2">
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)